core/src/main/java/org/apache/calcite/config/CalciteConnectionProperty.java

```diff
@@ -159,7 +159,10 @@
 
   /** Whether to make create implicit functions if functions do not exist
    * in the operator table, default false. */
-  LENIENT_OPERATOR_LOOKUP("lenientOperatorLookup", Type.BOOLEAN, false, false);
+  LENIENT_OPERATOR_LOOKUP("lenientOperatorLookup", Type.BOOLEAN, false, false),
+
+  /** Whether to enable top-down optimization in Volcano planner. */
+  TOPDOWN_OPT("topDownOpt", Type.BOOLEAN, false, false);
```

Comment:
Do you think it is OK to use the CalciteSystemProperty.TOPDOWN_OPT.value() as the default value?

Reply:
This is a good idea. I made this change. Also see comments above about further discussions.